### ISSUE TITLE
Feature Request - Entry Field to Select StartFromNodeID

### ISSUE THREAD

**Issue by nevsie** [am-impact/amnav#49]
In otherwords, a field with the ability to select which node to start from. Currently we only have a field for the whole menu, but just a sub part of that menu would be awesome.

**Comment by johnniefp** [comment]
Yes, I'd second this. It would make further sub level navigations possible if we could specify a node.

**Comment by johnniefp** [comment]
...hang on, just seen the param *startFromId*
Any chance of an example usage in the docs? (my assumptions on usage failed.)

**Comment by nevsie** [comment]
Yes this works in code. I was referring to a field in the entry edit page back end, allowing a client to select sub menu, and choose the child limb to display there... Just a thought!




On Sat, Sep 19, 2015 at 2:45 AM -0700, "johnniefp" <notifications@github.com> wrote:
...hang on, just seen the param *startFromId*
Any chance of an example usage in the docs? (my assumptions on usage failed.)

---
